molecule/default/tests/test_cartridge.py

```diff
@@ -17,7 +16,13 @@
 APP_NAME = 'myapp'
 HOSTS_PATH = os.path.join('molecule', 'default', 'hosts.yml')
 
+inventory = InventoryManager(loader=DataLoader(), sources=HOSTS_PATH)
+variable_manager = VariableManager(loader=DataLoader(), inventory=inventory)
+
+cluster_cookie = inventory.groups['cluster'].get_vars()['cartridge_cluster_cookie']
+
 __authorized_session = None
+__configured_instances = None
```

Comment:
Why do some variables start with "__" and some don't? All of them are not used in other modules, is it right?

Reply:
This is variables used on lazy-initialization.